src/chatd.h

```diff
@@ -1439,6 +1458,12 @@ class Client : public karere::DeleteTrackable
     promise::Promise<void> sendKeepalive();
     void sendEcho();
 
+    /** Handler of the timeout for retention history checks */
+    megaHandle mRetentionTimer;
+
+    /** Period in seconds until next check of retention history for all chats, or zero (disabled) */
+    uint32_t mRetentionCheckPeriod;
```

Comment:
With the last change it's not a period, it's the time stamp for next retention time. Probably other name improve the readability, per example mNextRetentionTime